app/code/Magento/MediaStorage/Model/File/Validator/NotProtectedExtension.php

```diff
@@ -93,7 +96,7 @@ protected function _initProtectedFileExtensions()
      * Return list with protected file extensions
      *
      * @param \Magento\Store\Model\Store|string|int $store
-     * @return string|string[]
+     * @return mixed
```

Comment:
`mixed` must not be used, please specify exact types.

Reply:
Ok, no problem. The underlaying function is returning "mixed" according to its doc type. But i will change it to "string|null".